src/components/scenes/NotificationScene.tsx

```diff
@@ -19,21 +19,23 @@ interface OwnProps {
 
 type Props = OwnProps
 
+type NotificationSettingToggleSetting = keyof NotificationSettings
+
 export const NotificationScene = (props: Props) => {
   const { navigation } = props
   const dispatch = useDispatch()
 
-  const settings = useSelector(state => state.priceChangeNotifications)
+  const settings = useSelector(state => state.notificationSettings)
   const account = useSelector(state => state.core.account)
 
   const currencyConfigs = useWatch(account, 'currencyConfig')
 
-  const toggleNotifications = useHandler(async () => {
+  const handlePressToggleSetting = useHandler(async (toggleSetting: NotificationSettingToggleSetting) => {
```

Comment:
There is no point doing `useHander` here since you destroy the optimization by passing inline closures to the `SettingsSwitchRow` components. Maybe make two handlers, each hard-coded for one setting? Or just ditch the `useHandler` wrapper at this point?

Reply:
The useHandler is because the handler is a dep for the useMemo. The only other alternative to which makes eslint happy is to remove the useMemo and inline the JSX (which always seemed cleaner to me). I'll add as a new commit to either accept or reject in re-review.